packages/core/src/components/Dialog/__stories__/Dialog.stories.js

```diff
@@ -19,14 +20,30 @@ const metaSettings = createStoryMetaSettingsDecorator({
   actionPropsArray: [] // List the component's actions here
 });
 
+const showHideArgTypes = {
```

Comment:
BTW have you tried using the `enumPropNamesArray`?

Reply:
yes, it is not doing anything that helps with the readability :(